packages/framework/src/Support/ReadingTime.php

```diff
@@ -81,7 +81,7 @@ public function getSecondsOver(): int
 
     public function getFormatted(string $format = '%dmin, %dsec'): string
     {
-        return sprintf($format, $this->getMinutes(), $this->getSecondsOver());
+        return sprintf($format, $this->getMinutes() ?: 1, $this->getMinutes() < 1 ? 0 : $this->getSecondsOver());
     }
 
     /** @param  \Closure(int, int): string $closure The closure will receive the minutes and seconds as integers and should return a string. */
```

Comment:
Am not adding any rounding to closure format as it should be as "pure" as possible since I assume anyone using this is because they want to run very custom logic.